api/urls.py

```diff
@@ -4,6 +4,8 @@
 
 urlpatterns = [
     path('banks/', views.BankListAPIView.as_view(), name='banks_api'),
-    path('user-token/', views.CreateUserTokenAPIView.as_view(), name='user_token_api'),
-    path('accounts/', views.AuthorizedUserAccountsAPIView.as_view(), name='user-accounts_api'),
+    path('bank/create/', views.CreateBankAPIView.as_view(), name='create_bank_api'),
+    path('user-token/', views.TokenObtainAPIView.as_view(), name='user_token_api'),
+    path('accounts/', views.AccountListCreateAPIView.as_view(), name='account-list-create_api'),
+    path('account/<int:pk>/', views.AccountRetrieveUpdateDestroyAPIView.as_view(), name='account-RUD_api'),
```

Comment:
Follow a consistent convention for url path with lower case with underscores `.., name=account_list`